Cargo.toml

```diff
@@ -66,6 +66,7 @@ once_cell = "1.17.1"
 sha2 = "0.10"
 serde = "1"
 serde_json = "1"
+metrics = "0.19.0"
```

Comment:
Could add a github issue to bump this once the ahash forced requirement is figured out // once penumbra updates.